[2024-06-07T06:43:31.264+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_toll_data_16.load_to_database manual__2024-06-07T06:43:28.025898+00:00 [queued]>
[2024-06-07T06:43:31.303+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_toll_data_16.load_to_database manual__2024-06-07T06:43:28.025898+00:00 [queued]>
[2024-06-07T06:43:31.305+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2024-06-07T06:43:31.357+0000] {taskinstance.py:1350} INFO - Executing <Task(_PythonDecoratedOperator): load_to_database> on 2024-06-07 06:43:28.025898+00:00
[2024-06-07T06:43:31.369+0000] {standard_task_runner.py:57} INFO - Started process 219 to run task
[2024-06-07T06:43:31.376+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ETL_toll_data_16', 'load_to_database', 'manual__2024-06-07T06:43:28.025898+00:00', '--job-id', '344', '--raw', '--subdir', 'DAGS_FOLDER/ETL_toll_data.py', '--cfg-path', '/tmp/tmpxdubib3m']
[2024-06-07T06:43:31.385+0000] {standard_task_runner.py:85} INFO - Job 344: Subtask load_to_database
[2024-06-07T06:43:31.554+0000] {task_command.py:410} INFO - Running <TaskInstance: ETL_toll_data_16.load_to_database manual__2024-06-07T06:43:28.025898+00:00 [running]> on host f8b9d10836c7
[2024-06-07T06:43:32.225+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='naveedcodes@gmail.com' AIRFLOW_CTX_DAG_OWNER='naveed' AIRFLOW_CTX_DAG_ID='ETL_toll_data_16' AIRFLOW_CTX_TASK_ID='load_to_database' AIRFLOW_CTX_EXECUTION_DATE='2024-06-07T06:43:28.025898+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-07T06:43:28.025898+00:00'
[2024-06-07T06:43:32.238+0000] {logging_mixin.py:149} INFO - database connected successfully
[2024-06-07T06:43:32.240+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-06-07T06:43:32.265+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=ETL_toll_data_16, task_id=load_to_database, execution_date=20240607T064328, start_date=20240607T064331, end_date=20240607T064332
[2024-06-07T06:43:32.304+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2024-06-07T06:43:32.360+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
